src/display.cpp

```diff
@@ -105,16 +105,41 @@ void setStatusBar(Gamepad *gamepad)
 	switch (gamepad->options.dpadMode)
 	{
 
-		case DPAD_MODE_DIGITAL:      statusBar += "         DPAD"; break;
-		case DPAD_MODE_LEFT_ANALOG:  statusBar += "         LEFT"; break;
-		case DPAD_MODE_RIGHT_ANALOG: statusBar += "        RIGHT"; break;
+		case DPAD_MODE_DIGITAL:      statusBar += "      DPAD"; break;
+		case DPAD_MODE_LEFT_ANALOG:  statusBar += "      LEFT"; break;
+		case DPAD_MODE_RIGHT_ANALOG: statusBar += "     RIGHT"; break;
 	}
 
 	switch (gamepad->options.socdMode)
 	{
 		case SOCD_MODE_NEUTRAL:               statusBar += "-N"; break;
 		case SOCD_MODE_UP_PRIORITY:           statusBar += "-U"; break;
 		case SOCD_MODE_SECOND_INPUT_PRIORITY: statusBar += "-L"; break;
+		case SOCD_MODE_X_NEUTRAL_Y_NEUTRAL:   statusBar += "-XNYN"; break;
```

Comment:
The current screen layout only leaves two characters for the SOCD mode.  Do we want to keep descriptions or could we replace them with `01`-`24` and leave the main three as `-N`, `-U`, `-L`?